src/logic/calculate.js

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line consistent-return
```

Comment:
- [ ] We should avoid disabling Eslint rules.  Instead, we need to make our code adequate with the specific rule. We can learn more about this specific rule [here](https://eslint.org/docs/rules/consistent-return)